src/main/docs/guide/kubernetes-client-openapi/kubernetes-client-openapi-config-client.adoc

```diff
@@ -93,6 +93,18 @@ kubernetes:
 
 Note that on the resulting config maps, you can still further filter them with includes/excludes properties.
 
+In case if you want to terminate the service startup when the service is not able to read config maps because of any
+unexpected exception, you can enable `terminate-startup-on-exception` property:
+
+[source,yaml]
```

Comment:
replace with `[configuration]` macro

Reply:
Went through all kubernetes doc files and modified everywhere